encord/http/constants.py

```diff
@@ -0,0 +1,20 @@
+from dataclasses import dataclass
+
+DEFAULT_MAX_RETRIES = 5
+DEFAULT_BACKOFF_FACTOR = 0.1
+
+
+@dataclass
+class RequestsSettings:
+    """
+    The settings for all outgoing network requests. These apply for each individual request.
+    """
+
+    max_retries: int = DEFAULT_MAX_RETRIES
+    """Number of allowed retries when a request is sent."""
+
+    backoff_factor: float = DEFAULT_BACKOFF_FACTOR
```

Comment:
I couldn't quickly find the "correct nomenclature" for this, but really the `factor` here is `2` and this value should be called `wait` or `initial_wait`.

Reply:
https://urllib3.readthedocs.io/en/stable/reference/urllib3.util.html uses the same definition. Seems to be a disagreement in Python land